### PR TITLE
added screenshots of the game

### PR DESCRIPTION
**Pull request by pfominftw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,19 @@ chemtrail-distribution system based on flight paths supplied by the user and the
...
 energy cost in Joules, as well as an optional visual representation on a map and the density of the gas at ground level. 
 The threshold gas level for visualization on the ground, as well as the plane height, should be specifiable by the user.
+
+Upon starting the game you will see a welcome screen:
+
+![welcome screen](mustards_game/screenshots/welcome_screen.png)
+
+Start game screen:
+
+![start game screen](mustards_game/screenshots/start_screen.png)
```
Comment (beliaev-maksim) [inline README.md:17]: do not think we need it
let's add another screenshot of the game itself, where more area is covered

Comment (pfominftw) [reply README.md:17]: more area is covered by the gas you mean?

Comment (beliaev-maksim) [reply README.md:17]: yes

**Comment by beliaev-maksim** [comment]
@pfominftw 
and now the same but on the latest master and maybe changing altitude :D

**Review by beliaev-maksim — CHANGES_REQUESTED** [review]
@pfominftw
and now the same but on the latest master and maybe changing altitude :D

File: README.md [hunk 2]
```diff
@@ -7,3 +7,15 @@ chemtrail-distribution system based on flight paths supplied by the user and the
...
 energy cost in Joules, as well as an optional visual representation on a map and the density of the gas at ground level. 
 The threshold gas level for visualization on the ground, as well as the plane height, should be specifiable by the user.
+
+Upon starting the game you will see a welcome screen:
+
+![welcome screen](mustards_game/screenshots/welcome_screen.png)
+
+Main game window:
+
+![main game window](mustards_game/screenshots/game2.png)
```
Comment (beliaev-maksim) [inline README.md:17]: where is game.png?

Comment (pfominftw) [reply README.md:17]: do we really need two screenshots?

Comment (beliaev-maksim) [reply README.md:17]: why not?

Comment (pfominftw) [reply README.md:18]: done

**Pull request merged**